lib/rules/no-tracked-properties-from-args.js

```diff
@@ -12,9 +12,9 @@ function visitClassPropertyOrPropertyDefinition(node, context, trackedImportName
     isClassPropertyOrPropertyDefinitionWithDecorator(node, trackedImportName);
 
   const hasThisArgsValue =
-    node.value.object?.type === 'MemberExpression' &&
-    node.value.object?.object.type === 'ThisExpression' &&
-    node.value.object?.property.name === 'args';
+    node.value?.object?.type === 'MemberExpression' &&
```

Comment:
Normally, it's better for us to be checking the type of the nodes we're dealing with, instead of just checking that various properties exist. Can we check the type on `node` and `node.value` before attempting to use them?

Reply:
👌

Reply:
Some really useful helpers and examples already built-ini I just needed to track down, thanks!